src/libs/CardUtils.ts

```diff
@@ -37,6 +37,18 @@ function isExpensifyCard(cardID: number) {
     return card.bank === CONST.EXPENSIFY_CARD.BANK;
 }
 
+/**
+ * @param cardID
```

Comment:
We can add a comment as `isCardListCard` is a bit tricky to understand from name

Reply:
renamed this!